comet/comet_test.go

```diff
@@ -116,13 +125,24 @@ func TestStatus(t *testing.T) {
 }
 
 func TestBroadcastTx(t *testing.T) {
-	chainID := "0"
-	app := testapp.NewTest(t, chainID)
+	app := testapp.NewTest(t, chainID.String())
```

Comment:
This is a common pattern across every test that needs to sign transactions with `testapp.ToTx`. We need to call `InitChain` before we fetch the application context, and then we get our pre-funded test account with `app.TestAccount`.

It'd probably be good to encapsulate this setup logic in a helper function (perhaps inside of `testapp.NewTest` itself), so we can avoid repeating it across each test.